src/containers/paper-canvas.jsx

```diff
@@ -82,14 +82,36 @@ class PaperCanvas extends React.Component {
         }
     }
     convertToBitmap () {
-        this.props.clearSelectedItems();
-        const raster = paper.project.activeLayer.rasterize(72, false /* insert */);
-        raster.onLoad = function () {
-            const subCanvas = raster.canvas;
-            getRaster().drawImage(subCanvas, raster.bounds.topLeft);
-            paper.project.activeLayer.removeChildren();
-            this.props.onUpdateSvg();
+        // @todo if the active layer contains only rasters, drawing them directly to the raster layer
+        // would be more efficient.
+        // Export svg
+        const guideLayers = hideGuideLayers(true /* includeRaster */);
+        const bounds = paper.project.activeLayer.bounds;
+        const svg = paper.project.exportSVG({
+            bounds: 'content',
+            matrix: new paper.Matrix().translate(-bounds.x, -bounds.y)
+        });
+        showGuideLayers(guideLayers);
+        
+        // Get rid of anti-aliasing
+        // @todo get crisp text?
+        svg.setAttribute('shape-rendering', 'optimizeSpeed');
```

Comment:
How does it look with `crispEdges` instead of `optimizeSpeed`? It seems like `crispEdges` is a better semantic match for the intent here, but I'm not sure of the actual behavior.

I did a quick test in Chrome by loading [the Scratch Cat SVG](https://cdn.assets.scratch.mit.edu/internalapi/asset/09dc888b0b7df19f70d81588ae73420e.svg/get/) directly then messing with the SVG in the dev console, and it looks the same with `crispEdges` or `optimizeSpeed`. The [MDN article on `shape-rendering`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/shape-rendering) seems to imply that it could vary across browsers, though.

Reply:
I think you're right that crisp edges seems more likely to work.

These specs sure use the word "might" a lot...